classes/Category.php

```diff
@@ -645,9 +651,6 @@ public function getProducts($id_lang, $p, $n, $order_by = null, $order_way = nul
 		if ($order_by == 'price')
 			$order_by = 'orderprice';
 
-		if (!Validate::isBool($active) || !Validate::isOrderBy($order_by) || !Validate::isOrderWay($order_way))
```

Comment:
- I think it's redundant (and also too strict): `$active` var is only used in ternary statement in lines 662, 696 and not inserted into SQL itself. The value is only evaluated as bool, we then can delete the unnecessary `die()` statement.